README.md

```diff
@@ -27,43 +27,65 @@ Our Task Tracker application defines four *Application Roles*:
+These application roles are defined in the [Azure Management Portal](https://manage.windowsazure.com/) on the application registration page.  When a user signs into the application, AAD emits a Role Claim for each role that has been granted based on the user and their group membership.  Assignment of users and groups to roles can be done through the portal's UI, or programmatically using the [AAD Graph API](http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx).  In this sample, application role management is done through the Azure Portal.
 
 Using RBAC with Application Roles and Role Claims, this application securely enforces authorization policies with minimal effort on the part of the developer.
 
 NOTE: Role claims are not currently emitted in SAML tokens, only JWTs (see issue #1).
 
 NOTE: Role claims are not currently emitted for guest users in a tenant (see issue #2).
 
-## How To Run The Sample as a MultiTenant App
+## How to build the sample?
 
+This sample is already registered in a Microsoft tenant as a multi-tenant application. Therefore you can run it in two different ways, depending on your business needs:
+- as a Multi-Tenant app (case 1). This requires little changes to the application itself, but there are contraints on the type of user.
+- as a Single-Tenant app (case 2).
+
+For more details about when you want to use a single tenant or a multi-tenant application, see the "configuring multi-tenant applications" paragraph of [Integrating applications with Azure Active Directory](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications)
+
+
+### Prerequisites
 To run this sample you will need:
-- Visual Studio 2013
+- Visual Studio 2013. You can also use more recent versions of Visual Studio (we explain below the line of configuration to change)
 - An Internet connection
 - An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, please see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/) 
-- A user account in your Azure AD tenant. This sample will not work with a Microsoft account, so if you signed in to the Azure portal with a Microsoft account and have never created a user account in your directory before, you need to do that now.
+- A user account in your Azure AD tenant. In the first case (run as a multi-tenant application), this sample will **not** work with **a Microsoft account**, so if you signed in to the Azure portal with a Microsoft account and have never created a user account in your directory before, you need to do that now. 

```

Comment:
Providing more guidance on how to create a user in the tenant, as this can be confusing for new AAD users.